index.js

```diff
@@ -178,3 +199,12 @@ function isString (str) {
 function isNumber (nr) {
   return typeof nr === 'number'
 }
+
+function setDependencies (jsonSchema, dependencies) {
+  const formattedDeps = {}
+  for (const prop in dependencies) {
+    const value = dependencies[prop]
+    formattedDeps[prop] = isString(value) ? [value] : value
+  }
+  jsonSchema.dependencies = formattedDeps
```

Comment:
Oh wow, I didn't know this is possible now.
And `ajv` also already supports it. ❤️